setup.py

```
@@ -3,24 +3,19 @@
 
-with open('README.md', 'r') as readme_file:
-    long_description = readme_file.read()
 
 setuptools.setup(
     name='PySnooper',
     version='0.0.1',
     author='Ram Rachum',
     author_email='ram@rachum.com',
     description="A poor man's debugger for Python.",
-    long_description=long_description,
+    long_description=open('README.md', 'r').read(),
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)